com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs

```diff
@@ -113,15 +113,12 @@ public static string GetName(this CameraRenderType type)
     [ImageEffectAllowedInSceneView]
     [MovedFrom("UnityEngine.Rendering.LWRP")] public class UniversalAdditionalCameraData : MonoBehaviour, ISerializationCallbackReceiver
     {
-        [Tooltip("If enabled shadows will render for this camera.")]
```

Comment:
Why are we removing tooltips?

Reply:
Because the property and Style is dealing with the Tool Tip.